images/scripts/fedora.setup

```diff
@@ -22,6 +22,9 @@ if [ "$IMAGE" != fedora-31 ]; then
 fi
 df -h /
 
+# make libpwquality less aggressive, so that our "foobar" password works
+printf 'dictcheck = 0\nminlen = 8\n' >> /etc/security/pwquality.conf
```

Comment:
still fails the same, maybe because `len("foobar")=6`?

Reply:
Yeah, of course that was meant to be a 6.. Fixing in #48 first